profile/rwahs.xml

```diff
@@ -931,6 +1053,8 @@
       <typeRestrictions>
         <restriction code="r1">
           <table>ca_objects</table>
+          <type>museum</type>
+          <includeSubtypes>true</includeSubtypes>
```

Comment:
All attributes on objects now have `<type></type>` included (except for those documented in the tests below)